src/app/script/security.cpp

```diff
@@ -185,6 +191,24 @@ int secure_os_execute(lua_State* L)
   return replaced_functions[os_execute].origfunc(L);
 }
 
+int secure_os_remove(lua_State* L)
+{
+  const char* cmd = luaL_checkstring(L, 1);
```

Comment:
I would suggest changing this line for something like:

```suggestion
  std::string absFilename = base::get_absolute_path(luaL_checkstring(L, 1));
  lua_pop(L, 1);
  lua_pushstring(L, absFilename.c_str());
```
And then adjust the following lines to use the new variable.

This change is to improve the paths that are reported by errors when passing relative paths to the os.remove function.

These changes should be applied to the secure_os_rename() function as well.

Reply:
Did the changes and noticed that for `os.rename` we'd have to check both source & destination path as well - additionally I added a QOL feature to `os.remove` (it was failing to delete empty directories with a "access denied" on Windows, and from googling it seems to be common that people just use `os.execute` but that's a whole different permission)